sql/MyDataAccess.java

```diff
@@ -0,0 +1,114 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class MyDataAccess {
+
+	private String _usuario="root";
+	private String _pwd= "greiscol";
+	private static String _bd="high-school";
+	static String _url = "jdbc:mysql://localhost/"+_bd;
+	private Connection conn = null;
+	
+	public MyDataAccess() {
+		
+		try{
```

Comment:
[High]: Move the DB connection logic from constructor method. It is advisable to use a pool connection to provide connections.